custom-routes-vis/README.md

```diff
@@ -7,8 +7,9 @@ It utilizes the Maps API and Java Servlets as the backend.
 2. Deploy to custom-routes-visualization.appspot.com with
 `mvn clean package appengine:deploy`
 
-Before running: Run `export DIRECTIONS_API_KEY=apiKey` in the terminal to set the
-environment variable.
-Before deploying: Change line 50 of GetDirectionsServlet.java to set the actual
-Directions API key in the code, because the environment variable cannot be accessed
+Before running: Run `export DIRECTIONS_API_KEY=apiKey` and `export CUSTOM_ROUTES_API_KEY=apiKey` 
```

Comment:
I thought they would input the custom routes key on their own? Like on the website. So I don't think this should be here because we should not provide a default custom routes API key for people to use.

Reply:
Zihan said he wants it there

Reply:
Okay I guess.